model/tensorflow_hub/examples/tfhub_text_classifier/textclassifier.py

```diff
@@ -0,0 +1,23 @@
+from dffml import CSVSource, Features, DefFeature
+from dffml.noasync import train, accuracy, predict
+from dffml_model_tensorflow_hub.text_classifier import TextClassificationModel
+
+model = TextClassificationModel(
+    features=Features(DefFeature("Sentence", str, 1)),
+    predict=DefFeature("Sentiment", int, 1),
+    classifications=[0, 1, 2],
+    clstype=int,
+)
+# train the model
+train(model, "train.csv")
```

Comment:
@pdxjohnny I've followed the format used in other examples. However, I'm having trouble running this and the other examples, that you mentioned for reference, locally. It seems that it doesn't find the "train.csv" file. While running the test, it says "no records to train on". So I suppose it is the same problem that I face locally that the tests are facing.

Reply:
But this would also mean that the other examples for scratch etc. are also not working which seemed unlikely so I wanted to know if there is something else bugging it, before going about it.

Reply:
There's a missmatch on the capitalization of the headers in the csv files vs. the features defined

Reply:
It's saying there are not records to train on because there are no records with matching features. Because the features are not the same capitalization

Reply:
Ah! I see. Thank you!